### ISSUE TITLE
Not possible to 'save/save as' a file (branch main)

### ISSUE THREAD

**Issue by pissard** [braillerap/DesktopBrailleRAP#1]
With an ubuntu version is not possible to 'save/save as' a file. We have this error message on the console

Traceback (most recent call last):
  File "/home/pissard/Dev/Hli/braillerap/DesktopBrailleRAP/venv/lib/python3.11/site-packages/webview/util.py", line 220, in _call
    result = func(*func_params)
             ^^^^^^^^^^^^^^^^^^
  File "/home/pissard/Dev/Hli/braillerap/DesktopBrailleRAP/desktopbraillerap.py", line 142, in saveas_file
    with open(filename, "w", encoding="utf8") as inf:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: expected str, bytes or os.PathLike object, not tuple

**Comment by pissard** [comment]
replace the code

```
  if fname == "" or fname == None:
                return
            filename = fname
```

with

```
if fname:
            filename = fname[0]
        else:
            return
```

**Comment by crocsg** [comment]
fix proposed in commit 2dc086cb841050916cb30e94a8e4244d3bc1b370
